### PR TITLE
Update flw-onboarding-wizard.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated navigation to play the video

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9841****

File: microsoft-365/frontline/flw-onboarding-wizard.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Check out this short video for an overview of how to run the wizard to onboard y
 
 ## Run the wizard
 
-1. In the left navigation of the [Microsoft 365 admin center](https://admin.microsoft.com/), choose **Setup**. Go to the **Apps and email** section, and then under **Get your frontline workforce up and running**, select **View**. Here, you can learn more about the capabilities that Microsoft 365 for frontline workers offers.
+1. In the left navigation of the [Microsoft 365 admin center](https://admin.microsoft.com/), choose **Setup**. Go to the **Apps and email** section, and then under **Get your frontline workforce up and running**, Play the **Video** to learn more about the capabilities that Microsoft 365 for frontline workers offers.
```
Comment (JohanFreelancer9) [inline microsoft-365/frontline/flw-onboarding-wizard.md]:
```suggestion
1. In the left navigation of the [Microsoft 365 admin center](https://admin.microsoft.com/), choose **Setup**. Go to the **Apps and email** section, and then under **Get your frontline workforce up and running**, select **Video** to learn more about the capabilities that Microsoft 365 for frontline workers offers.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@LanaChin Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@LanaChin Gentle reminder, please merge this PR. Thanks

**Comment by samanro** [comment]
Updating in separate PR: https://github.com/MicrosoftDocs/microsoft-365-docs-pr/pull/18576

**Pull request closed without merging**